pytools/graph.py

```diff
@@ -236,6 +237,75 @@ def compute_topological_order(graph):
 
     return list(reversed(reverse_order))
 
+
+def compute_topological_order_v2(graph, key=None):
+    """Compute a toplogical order of nodes in a directed graph.
```

Comment:
Could you link to a source for Kahn's algorithm/state in the code that that's what's being used?